frontend/src/lib-customizations/espoo/citizen.tsx

```diff
@@ -2,20 +2,129 @@
-    sv: {},
-    en: {}
+    fi: {
+      income: {
+        assure: (
+          <MultiLineCheckboxLabel>
+            <span>
+              Vakuutan antamani tiedot oikeiksi ja olen tutustunut
+              asiakasmaksutiedotteeseen:
+            </span>
+            <a
+              href="https://www.espoo.fi/fi/artikkelit/lasten-varhaiskasvatuksesta-perittavat-maksut-182023-alkaen"
+              target="_blank"
+              rel="noreferrer"
+            >
+              Lasten varhaiskasvatuksesta perittävät maksut 1.8.2023 alkaen |
+              Espoon kaupunki
+            </a>
+          </MultiLineCheckboxLabel>
+        ),
+        incomeType: {
+          description: (
+            <>
+              Jos olet yrittäjä, mutta sinulla on myös muita tuloja, valitse
+              sekä <strong>Yrittäjän tulotiedot</strong>, että{' '}
+              <strong>Asiakasmaksun määritteleminen tulojen mukaan</strong>.
+            </>
+          ),
+          grossIncome: 'Maksun määritteleminen tulojen mukaan'
+        },
+        grossIncome: {
+          title: 'Tulotietojen täyttäminen',
+          estimate: 'Arvio palkkatuloistani (ennen veroja)'
+        }
+      }
+    },
+    sv: {
+      income: {
+        assure: (
+          <MultiLineCheckboxLabel>
+            <span>
+              Jag försäkrar att de uppgifter jag lämnat in är riktiga och jag
+              har bekantat mig med kundcirkuläret gällande avgifter för
+              småbarnspedagogik:{' '}
+              <a
+                href="https://www.espoo.fi/sv/artiklar/avgifter-smabarnspedagogik-fran-182023"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Avgifter för småbarnspedagogik från 1.8.2023 | Esbo stad
+                (espoo.fi)
+              </a>
+            </span>
+          </MultiLineCheckboxLabel>
+        ),
+        incomeType: {
+          description: (
+            <>
+              Om du är företagare men har också andra inkomster, välj både{' '}
+              <strong>Företagarens inkomstuppgifter</strong>, och{' '}
+              <strong>Fastställande av klientavgiften enligt inkomster</strong>.
+            </>
+          ),
+          grossIncome: 'Fastställande av avgiften enligt inkomster'
+        },
+        grossIncome: {
+          title: 'Att fylla i uppgifterna om inkomster',
+          estimate: 'Uppskattning av mina bruttolön (före skatt)'
+        }
+      }
+    },
+    en: {
+      income: {
+        assure: (
+          <MultiLineCheckboxLabel>
+            <span>
+              I confirm that the information I have provided is accurate, and I
+              have reviewed the customer fee information:
+            </span>
+            <a
+              href="https://www.espoo.fi/en/articles/early-education-fees-1-august-2023"
+              target="_blank"
+              rel="noreferrer"
+            >
+              Early education fees as of 1 august 2023 | City of Espoo
```

Comment:
august -> August

Reply:
I guess you can't trust even official translations :D